addon/components/attach-popover.js

```diff
@@ -349,7 +350,7 @@ export default Component.extend({
 
   _removeEventListeners() {
     Object.keys(this._hideListenersOnDocumentByEvent).forEach((eventType) => {
-      document.removeEventListener(eventType, this._hideListenersOnDocumentByEvent[eventType]);
+      document.removeEventListener(eventType, this._hideListenersOnDocumentByEvent[eventType], this.get('useCapture'));
```

Comment:
Wouldn't this cause a problem if `useCapture` changes before an event listener is removed?

Reply:
Yes it would - this PR is assuming that the `useCapture` value is immutable. I think requiring it to be immutable is reasonable, but this could be either directly enforced or made more explicit. Can you think of a good use case for changing it dynamically? In my app where this is being used I'm setting it to true in `config/environment.js`.

Reply:
I think setting a guard on changing `useCapture` is a reasonable-enough approach. Let's throw an error if one attempts to do so

Reply:
3d24fc5 adds a warning that triggers when `useCapture` is mutated.